### PR TITLE
put legend below map on mobile and keep in map for larger screens

### PR DESCRIPTION
**Pull request by tanyaparker**
**Desktop**
image

**Mobile**
image

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/public/index.haml [hunk 1]
```diff
@@ -128,11 +128,6 @@ social:
   %br/
 
   .row
-    .col-xs-12
-      %div{style: 'text-align: center; margin: 3em 0;'}
-        %h2=hoc_s(:highlights_hoc_2014_heading)
```
Comment (tanyaparker) [inline pegasus/sites.v3/hourofcode.com/public/index.haml:133]: This string doesn't exist and no header is being used above the images, so I just removed it.

**Comment by tanyaparker** [comment]
@breville fyi -- if you know of a better method let me know.

File: pegasus/sites.v3/hourofcode.com/views/map.haml [hunk 2]
```diff
@@ -32,6 +32,12 @@
 .row
   .col-xs-10.col-sm-12
     #gmap.full-width
+#belowmaplegend.legend
+  .legend-title= hoc_s(:map_legend_title)
+  .color.turquoise
+  = hoc_s(:map_legend_hoc_events)
```
Comment (caleybrock) [inline pegasus/sites.v3/hourofcode.com/views/map.haml:38]: are these strings in a separate pr?

**Comment by caleybrock** [comment]
css lgtm, as long as those strings already exist

**Comment by tanyaparker** [comment]
yup strings already on staging

**Pull request merged**